### PR TITLE
Fit data extract function from log and test cases

### PR DESCRIPTION
**Pull request by SAYANTANDE**
The locations of the files might be messed up, I wasn't sure where to put them. I think the test file needs some modification, the parse log function needs to be placed somewhere so it can be called/defined.

### CODE REVIEW

File: src/parse_log.jl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+function parse_log(log)
+    models_dict = Dict{String, Vector{Dict{String, Any}}}()
+
+    for (idx, entry) in enumerate(log)
+        if entry.action.funct == :modelfit
+            model_name = entry.info.model_name
+            model_params = entry.info.model_params
+            error = entry.info.error
+
+            model_entry = Dict("params" => model_params, "error" => error, "index" => idx)
```
Comment (akabla) [inline src/parse_log.jl]: Make it a named tupple, please:
` model_entry = (params = model_params, info = info, index = idx) `

File: src/parse_log.jl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+"""
+function parse_log(log)
+    models_dict = Dict{String, Vector{Dict{String, Any}}}()
+
+    for (idx, entry) in enumerate(log)
+        if entry.action.funct == :modelfit
+            model_name = entry.info.model_name
+            model_params = entry.info.model_params
+            error = entry.info.error
```
Comment (akabla) [inline src/parse_log.jl]: Let's supply the whole info field. We can remove this.

**Review by SAYANTANDE — COMMENTED** [review]
I have added the extractfitdata function and test to respective rheodata.jl files and added the export of extractfitdata function to RHEOS.jl file in src

**Comment by akabla** [comment]
Will fix later.

**Pull request merged**